tutorials/notebooks/specutils-fit-line/specutils-fit-line1.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>Add link to list of all "loaders". https://specutils.readthedocs.io/en/stable/spectrum1d.html#list-of-loaders.</p><p> </p><p>Specutils has <a href="https://specutils.readthedocs.io/en/stable/spectrum1d.html#list-of-loaders" target="_blank">many loaders</a> for common spectra formats including one for SDSS spectra! </p><p> </p><p>Call it a "loader" instead of a reader to maintain consistency. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>